frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tabs/snapshot.ts

```diff
@@ -0,0 +1,97 @@
+import { yamlVMName } from '../../const/index';
+import { ProvisionSource } from '../../enums/provisionSource';
+import { testName } from '../../support';
+import { VirtualMachineData } from '../../types/vm';
+import { alertTitle, modalConfirm, modalCancel } from '../../view/selector';
+import * as snapshotView from '../../view/snapshot';
+import { tab } from '../../view/tab';
+import { virtualization } from '../../view/virtualization';
+import { vm } from '../../view/vm';
+
+const vmData: VirtualMachineData = {
+  name: `test-vm-snapshot-${testName}`,
+  description: 'rhel8 vm for snapshot',
+  namespace: testName,
+  template: 'Red Hat Enterprise Linux 8.0+ VM',
```

Comment:
Do we have strings anywhere for the templates? If not, we should probably add them.